sdk/identity/Azure.Identity/src/VisualStudioCodeCredential.cs

```diff
@@ -61,6 +61,11 @@ private async ValueTask<AccessToken> GetTokenImplAsync(TokenRequestContext reque
 
             try
             {
+                if (string.Equals(_tenantId, Constants.AdfsTenantId, StringComparison.Ordinal))
```

Comment:
Does case matter here?

Reply:
Probably in practice no, but since it's part of the Authority path, it is possible the service would treat this as case sensitive. So to err on the side of caution I was doing a case insensitive compare for the time being, and we can document that users should use the tenant id `"adfs"` all lowercase when authenticating with ADFS.